lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {
+  NxThreatIndicatorLegendProps,
+  nxThreatIndicatorLegendPropTypes,
+  ThreatProps
+} from './types';
+import { NxThreatIndicator } from '../..';
```

Comment:
Don't import from `index.ts` in the component files; that creates a circular dependency.  Instead import other components from their original source files.

Reply:
> Don't import from `index.ts` in the component files; that creates a circular dependency. Instead import other components from their original source files.

The import has been fixed:
69c908901ff125262248d4140217d33670a49ec4